src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundUserAcceptanceTest.java

```diff
@@ -19,18 +23,15 @@
  * the message from the inbound queue to be the same to the message which has been sent.
```

Comment:
Should we modify the comments because now they seem to be not relevant anymore?

Reply:
I've changed this slightly, let me know if it needs modifying more :-)

Reply:
Yes, seems good now. Thank you!